util/packaging/homebrew/chapel-main.rb

```diff
@@ -48,6 +34,10 @@ def llvm
   end
 
   def install
+    # Always detect Python used as dependency rather than needing aliased Python formula
+    python = "python3.10"
+    inreplace "util/config/find-python.sh", /^(for cmd in )(python3 )/, "\\1#{python} \\2"
```

Comment:
Interesting. It looks like this changes

```
for cmd in python3 python python2; do
```

into

```
for cmd in python3.10 python python2; do
```

is that right? IMO it would be better to change it to


```
for cmd in python3.10 python3 python python2; do
```

but I am OK with the current situation, assuming I understand it. It would be good to add a comment about the transformation it is trying to do here.

Reply:
Well, I will say that I did not write this code. This came out of the current formula published in the homebrew project.  I can add some additional comments here and this could definitely be a bug on their part.